helix-term/src/commands.rs

```diff
@@ -2819,6 +2818,13 @@ pub mod cmd {
             fun: theme,
             completer: Some(completers::theme),
         },
+        TypableCommand {
+            name: "theme-base16",
```

Comment:
I don't think this should be two different commands, just a single one that behaves differently depending on the config. For that to work, https://github.com/helix-editor/helix/pull/1134 needs to land first though, since it adds `cx` to completions